src/cartridges/app_adyen_SFRA/cartridge/client/default/js/paypalExpress.js

```diff
@@ -0,0 +1,225 @@
+const {
+  getPaymentMethods,
+  updateLoadedExpressMethods,
+  checkIfExpressMethodsAreReady,
+} = require('./commons');
+const helpers = require('./adyen_checkout/helpers');
+const { PAYPAL } = require('./constants');
+
+async function callPaymentFromComponent(data, component) {
+  try {
+    $.spinner().start();
+    const response = await fetch(window.makeExpressPaymentsCall, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    });
+    const { action, errorMessage = '' } = await response.json();
+    if (response.ok && action) {
+      component.handleAction(action);
+    } else {
+      throw new Error(errorMessage);
```

Comment:
I guess we are assuming an error message will be populated, but would't be better for debug something like: 
`throw new Error(errorMessage || 'Express payment action failed');`

Reply:
This applies to all Error handling in the code.

Reply:
This is a front-end part, so ideally we want to keep the logging as limited as possible there. If there is an error coming from the response we will throw it, otherwise we leave it empty.